app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -1,20 +1,32 @@
 import { useContext } from 'react';
+import { t } from '@18f/identity-i18n';
 import UploadContext from '../context/upload';
 import AnalyticsContext from '../context/analytics';
+import { FormError } from '../components/form-steps';
 
 /**
  * @typedef {import('../components/form-steps').FormStepComponentProps<V>} FormStepComponentProps
  * @template V
  */
 
+/**
+ * Non-breaking space (`&nbsp;`) represented as unicode escape sequence, which React will more
+ * happily tolerate than an HTML entity.
+ */
+const NBSP_UNICODE = '\u00A0';
+
 /**
  * An error representing a failure to complete encrypted upload of image.
  */
-export class BackgroundEncryptedUploadError extends Error {
+export class BackgroundEncryptedUploadError extends FormError {
   baseField = '';
 
   /** @type {string[]} */
   fields = [];
+
+  message = `${t('doc_auth.errors.upload_error')} ${t('errors.messages.try_again')
+    .split(' ')
+    .join(NBSP_UNICODE)}`;
```

Comment:
> would this be approximate the same if we did:

No, since that would create non-breaking spaces between every word in the error message. The idea with the NBSP is to make sure that "Please try again" stays **or** breaks onto a new line, but importantly it stays together. Applying it to the whole string would probably prevent any line breaks altogether (see before/after screenshots in #4773).

Maybe what we could do is something like:

```suggestion
  const tryAgain = t('errors.messages.try_again').split(' ').join(NBSP_UNICODE);
  message = `${t('doc_auth.errors.upload_error')} ${tryAgain}`;
```

?

Reply:
Ah! Yes, this new option makes a lot more sense